app/services/diagnostics/move_inspector.rb

```diff
@@ -195,6 +213,8 @@ def generate
             @output << "status:\t#{per.status}\n"
             @output << "created at:\t#{per.created_at}\n"
             @output << "updated at:\t#{per.updated_at}\n"
+            @output << "amended at:\t#{per.amended_at}\n"
```

Comment:
Would be good to also add `per.completed_at` here - that's also supported for youth risk assessments

Reply:
👍